contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,726 @@
+---
+title: "The most popular PostHog alternatives, compared"
+date: 2023-06-22
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+As an all-in-one platform, PostHog replaces _a lot_ of legacy tools. PostHog is great if you:
+
+1. Need quantitative _and_ qualitative insights into user behavior
+2. Want to use feature flags for rolling out new features, kill switches, and feature gating
```

Comment:
any specific reason we're calling out feature flags here and not any of the other features? Not sure why the emphasis on it.

I think ti would be good to call out all our core features here.